Modules/_ctypes/_ctypes.c

```diff
@@ -4054,6 +4059,8 @@ _init_pos_args(PyObject *self, PyTypeObject *type,
     }
 
     dict = PyType_stgdict((PyObject *)type);
+    if (dict == NULL)
```

Comment:
We've started to put curly braces around one-liners: https://www.python.org/dev/peps/pep-0007/#code-lay-out.